roles/openshift_default_storage_class/defaults/main.yml

```diff
@@ -12,6 +12,11 @@ openshift_storageclass_defaults:
     provisioner: gce-pd
     parameters:
       type: pd-standard
+  openstack:
+    name: standard
+    provisioner: cinder
+    parameters:
+      availability: nova
```

Comment:
I think that what I did in my PR is more generic, because I did not do any assumptions about the cinder availability zones. (e.g. one of the environments where I tested has no availability zone called nova)

Reply:
Yes, you are right.